pageserver/src/page_cache.rs

```diff
@@ -8,29 +8,32 @@
 
 use crate::restore_local_repo::restore_timeline;
 use crate::ZTimelineId;
-use crate::{walredo, PageServerConf};
+use crate::{walredo, PageServerConf, zenith_repo_dir};
 use anyhow::bail;
-use bytes::Bytes;
-use core::ops::Bound::Included;
+use bytes::{Buf, BufMut, Bytes, BytesMut};
 use crossbeam_channel::unbounded;
 use crossbeam_channel::{Receiver, Sender};
 use lazy_static::lazy_static;
 use log::*;
-use rand::Rng;
-use std::collections::{BTreeMap, HashMap};
+use std::cmp::min;
+use std::collections::HashMap;
 use std::sync::atomic::AtomicU64;
 use std::sync::atomic::Ordering;
 use std::sync::{Arc, Condvar, Mutex};
 use std::thread;
 use std::time::Duration;
 use std::{convert::TryInto, ops::AddAssign};
+use rocksdb::*;
```

Comment:
Can we please import only the symbols that are used?

Reply:
I think I changed my mind. The symbols needed from rocksdb include things named `Options`, `Direction`, and `IteratorMode`. Since those names don't communicate anything DB-related, I think the code would be more readable if we instead use `rocksdb::Options`, `rocksdb::Direction`, etc.